pandas/tests/frame/test_query_eval.py

```diff
@@ -165,7 +164,9 @@ def test_eval_resolvers_as_list(self):
 
 class TestDataFrameQueryWithMultiIndex(tm.TestCase):
 
-    def check_query_with_named_multiindex(self, parser, engine):
+    @pytest.mark.parametrize('parser', ['pandas'])
```

Comment:
I wonder if a comment here explaining why we're parametrizing instead of just hardcoding `parser` would be useful.